### PR TITLE
Runner in Server [CIRCLE-33129]

### PR DESCRIPTION
**Pull request by kkowalski**
# Description
Adding changes needed when setting up a Runner for Server.

# Reasons
https://circleci.atlassian.net/browse/CIRCLE-33129

### CODE REVIEW

File: jekyll/_cci2/runner-server.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+categories: [platforms]
+description: How to set up a Runner in Server
+order: 31
+version:
+- Server v3.x
+---
+
+## Authentication
+
+When creating namespaces. resource classes and tokens, the CLI needs to be configured to connect to the Server
```
Comment (thomasr8) [inline jekyll/_cci2/runner-server.md:14]:
```suggestion
When creating namespaces, resource classes and tokens, the CLI needs to be configured to connect to the Server 
```

File: jekyll/_cci2/runner-server.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+description: How to set up a Runner in Server
+order: 31
+version:
+- Server v3.x
+---
+
+## Authentication
+
+When creating namespaces. resource classes and tokens, the CLI needs to be configured to connect to the Server 
+deployment either via `--host HOSTNAME` and `--token TOKEN` flags or CLI's config file.
```
Comment (thomasr8) [inline jekyll/_cci2/runner-server.md:15]:
```suggestion
deployment either via `--host HOSTNAME` and `--token TOKEN` flags, or the CircleCI CLI's configuration file.
```

File: jekyll/_cci2/runner-server.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+####  Resource class example
+```plaintext
+circleci runner resource-class create <resource-class> <description> --host HOSTNAME --token TOKEN
+```
+
+## Configuration file
+
+When setting up a Runner the configuration file should include `host` property.
```
Comment (thomasr8) [inline jekyll/_cci2/runner-server.md:24]:
```suggestion
When setting up a Runner, the configuration file should include `host` property.
```

File: jekyll/_cci2/runner-server.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+runner:
+  name: RUNNER_NAME
+  command_prefix: ["sudo", "-niHu", "circleci", "--"]
+  working_directory: /opt/circleci/workdir/%s
+  cleanup_working_directory: true
+```
+
+## Version
+
+Specific server version works with specific runner version. The table below presents the mapping.
```
Comment (thomasr8) [inline jekyll/_cci2/runner-server.md:39]:
```suggestion
A specific server version works with a specific runner version. The table below presents the mapping.
```

**Review by thomasr8 — CHANGES_REQUESTED** [review]

**Comment by heug** [comment]
@thomasr8 could i get a heads up when this is ready for translation please? thank you!

**Review by glenjamin — COMMENTED** [review]
Execution content looks fine to me, should perhaps link to the rest of the cloud runner docs?

**Comment by kkowalski** [comment]
It will be under Runner section so customer should be able to to see other runner related docs in the tree next to this article

**Review by thomasr8 — APPROVED** [review]
Looks good!

**Comment by glenjamin** [comment]
I wonder if this needs a note to say it’s not available in 3.0 and to
contact us?

**Review by nsmith5 — APPROVED** [review]

File: jekyll/_data/sidenav.yml [hunk 5]
```diff
@@ -134,6 +134,8 @@ en:
           link: 2.0/runner-api/
         - name: Runner FAQs
           link: 2.0/runner-faqs/
+        - name: Runner in Server
+          link: 2.0/runner-server/
```
Comment (annabaker) [inline jekyll/_data/sidenav.yml:138]:
```suggestion
        - name: Runner in Server
          link: 2.0/runner-server/
        - name: Runner FAQs
          link: 2.0/runner-faqs/
```

Comment (annabaker) [reply jekyll/_data/sidenav.yml:138]: Looks good -- just rearranged the sidebar links so FAQ is at the bottom. This is fine to be merged now that Server 3 docs are public.

**Review by annabaker — APPROVED** [review]

**Comment by glenjamin** [comment]
The key thing to note here is that Runner isn't available in the current version of Server - it will only be available in special preview versions until the next release.

**Comment by kkowalski** [comment]
@annabaker could you add this note that @glenjamin mentioned ? I talked with @thomasr8 about it.

**Pull request merged**